src/search.ts

```diff
@@ -34,8 +34,14 @@ export interface Position {
 export async function search(content: string): Promise<SearchValue> {
   const tree = await acorn.parse(content, {locations: true});
   const nodeArr: Node[] = getRequireCalls(tree);
-  const result: SearchValue = getRequiredModules(nodeArr, iOModules);
-
+  const result: SearchValue = getRequiredModules(nodeArr);
+  const moduleMap = result.requiredModules;
+  moduleMap.forEach((val, key) => {
+    if (!ioModules.includes(key)) {
+      moduleMap.delete(key);
+    }
+  });
+  result.requiredModules = moduleMap;
```

Comment:
This line is unnecessary, as `moduleMap` and `result.requiredModules` point to the same object.